### PR TITLE
Added new duration metrics

### PR DESCRIPTION
**Pull request by Josca**

### CODE REVIEW

File: evaluator/metrics.go [hunk 1]
```diff
@@ -17,12 +17,12 @@ var (
 		Name:      "updates",
 	}, []string{"type"})
 
-	evaluationDuration = prometheus.NewHistogram(prometheus.HistogramOpts{
+	evaluationDuration = prometheus.NewHistogramVec(prometheus.HistogramOpts{
 		Namespace: "patchman_engine",
 		Subsystem: "evaluator",
 		Name:      "evaluation_duration_milliseconds",
 		Buckets:   []float64{.00001, .0001, .001, .01, .1, 1, 10, 100, 1000},
```
Comment (semtexzv) [inline evaluator/metrics.go]: We need the most resolution in the 1ms to ~10000ms range.

File: listener/delete.go [hunk 2]
```diff
@@ -4,9 +4,13 @@ import (
 	"app/base/database"
 	"app/base/mqueue"
 	"app/base/utils"
+	"time"
 )
 
 func deleteHandler(event mqueue.PlatformEvent) {
+	tStart := time.Now()
+	defer messageHandlingDuration.WithLabelValues("delete").Observe(time.Since(tStart).Seconds())
```
Comment (semtexzv) [inline listener/delete.go]: There is EventDelete const.

File: listener/metrics.go [hunk 3]
```diff
@@ -23,10 +23,16 @@ var (
 		Subsystem: "listener",
 		Name:      "kafka_message_received",
 	}, []string{"event", "type"})
+
+	messageHandlingDuration = prometheus.NewHistogramVec(prometheus.HistogramOpts{
+		Namespace: "patchman_engine",
+		Subsystem: "listener",
+		Name:      "kafka_message_handling_duration_seconds",
```
Comment (semtexzv) [inline listener/metrics.go]: Missing buckets, I'd also unify the names between listener and evaluator, something like `handle_duration`

Comment (Josca) [reply listener/metrics.go]: We can use default buckets here. This is the convention according to Prometheus docs to add duration_{unit} suffix. I wouldn't add "handle" word to evaluation duration metric name.

**Pull request merged**